<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlendWizard</class>
 <widget class="QWizard" name="BlendWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blend</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::AeroStyle</enum>
  </property>
  <widget class="QWizardPage" name="Blend1">
   <property name="title">
    <string>Blend Wizard</string>
   </property>
   <property name="subTitle">
    <string>Blend is ImageRaster utility to create professional looking thumbnail for your image. Press next to continue.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">0</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QWizardPage" name="Blend2">
   <property name="title">
    <string>Open Image for Thumbnail</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="browse"/>
    </item>
    <item>
     <widget class="QPushButton" name="browse_b">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="Blend3">
   <property name="title">
    <string>Crop</string>
   </property>
   <property name="subTitle">
    <string>Select part of the image that will be presented as thumbnail. Unnecessary part will be neglected. You can press right mouse button on image to reset selection.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">2</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="view1">
      <property name="cursor" stdset="0">
       <cursorShape>CrossCursor</cursorShape>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="Blend4">
   <property name="title">
    <string>Style &amp; Position Editor</string>
   </property>
   <property name="subTitle">
    <string>Here you can change your thumbnail appearance and add it to the scene. Note that you can't change thumbnail style after you close this wizard.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">3</string>
   </attribute>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Style:</string>
         </property>
         <property name="buddy">
          <cstring>style</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="style">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Circle (clip)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Box (clip)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Circle (strectch)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Box (stretch)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ellipse</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rectangle</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Thumbnail size:</string>
         </property>
         <property name="buddy">
          <cstring>thumbSize</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="thumbSize"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Anchor size:</string>
         </property>
         <property name="buddy">
          <cstring>anchorSize</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="anchorSize"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Border color:</string>
         </property>
         <property name="buddy">
          <cstring>color1</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ColorEditor" name="color1"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Outline color:</string>
         </property>
         <property name="buddy">
          <cstring>color2</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="ColorEditor" name="color2"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Border thickness:</string>
         </property>
         <property name="buddy">
          <cstring>lineweight</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="lineweight"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Text:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="text">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="view2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>250</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorEditor</class>
   <extends>QComboBox</extends>
   <header>coloreditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
